### PR TITLE
feat: trending page

### PR DESCRIPTION
**Pull request by tusharkandpal**
To check

- added `Trending` page
- `videoSlice.js` for reducer & async operations
- components added- `VideoCard`
- services includes `getCategories.js` & `getVideos`
- util for `getViews`

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks very well structured!! Good work @tusharkandpal 
The live preview is also looking good : https://deploy-preview-2--power-play.netlify.app/trending
With a proper loading message of Videos!!
Got a clear understanding of Redux Implementation used to develop these actions.
One query I got on the live preview is that while viewing this webpage a white space is  being developed at the bottom occupying a small portion while we can scroll through the whole trending section. So you can look onto that screen height adjustment issue!!

File: src/utils/getViews.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const getViews = (views) => {
```
Comment (sushil-mod) [inline src/utils/getViews.js:1]: I liked how you used to convert number views into k and m format

**Review by sushil-mod — APPROVED** [review]
well implemented redux, and also you created a separate file for service calls . A little bit of UI is not fixed as Roshan suggested. Rest is good

**Comment by tusharkandpal** [comment]
> The code looks very well structured!! Good work @tusharkandpal The live preview is also looking good : https://deploy-preview-2--power-play.netlify.app/trending With a proper loading message of Videos!! Got a clear understanding of Redux Implementation used to develop these actions. One query I got on the live preview is that while viewing this webpage a white space is being developed at the bottom occupying a small portion while we can scroll through the whole trending section. So you can look onto that screen height adjustment issue!!

Yeah ! I'm new to the Redux part too, so learning as I implement it. Thanks for the detailed feedback @dev-enforced

File: src/utils/getViews.js (hunk 1 above)
Comment (tusharkandpal) [reply src/utils/getViews.js:1]: > well implemented redux, and also you created a separate file for service calls . A little bit of UI is not fixed as Roshan suggested. Rest is good

Surprisingly, I don't see any UI issue on my end.

Thanks for the review @sushil-mod

**Pull request merged**